src/chi-vue/src/components/column-customization/ColumnCustomizationModalContent.tsx

```diff
@@ -11,19 +11,17 @@ import { _changeOrder } from '@/components/column-customization/utils';
 @Component
 export default class ColumnCustomizationContent extends Vue {
   @Prop() availableColumns?: DataTableColumn[];
-  @Prop() selectedLockedColumns?: DataTableColumn[];
   @Prop() selectedStandardColumns?: DataTableColumn[];
 
   key = 0;
   _availableColumns?: DataTableColumn[] = [];
-  _selectedLockedColumns?: DataTableColumn[] = [];
   _selectedStandardColumns?: DataTableColumn[] = [];
   _availableColumnsComponent?: AvailableColumns;
   _selectedColumnsComponent?: SelectedColumns;
+  _isSelectedColumnLocked = false;
```

Comment:
This name seems to be related to a *single* locked column. Is it be possible to have more than 1 selected locked columns at the same time. If so, this name is not good.